lib/ingreedy/dictionary_collection.rb

```diff
@@ -12,17 +12,25 @@ def []=(locale, attributes)
     end
 
     def current
+      fetch_dictionary(current_locale)
+    end
+
+    def current_locale
+      find_locale
+    end
+
+    private
+
+    def find_locale
       candidate_locales.each do |locale|
         if dictionary = fetch_dictionary(locale)
-          return dictionary
+          return locale
         end
       end
 
       raise "No dictionary found for locales: #{candidate_locales}"
     end
```

Comment:
> it would be better to add locale as a readonly attribute on Dictionary which is passed when the instance is initialized? 

Oh!! It's great.
Actually, I thought the logic here was odd myself. But I couldn't think of a good solution.😅